internal/core/ui/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+  <string name="recco_bookmarks_empty_state_title">Du hast noch keine Lesezeichen gesetzt. </string>
+  <string name="recco_bookmarks_title">Deine Bookmarks</string>
+  <string name="recco_continue_button">Weiter</string>
+  <string name="recco_dashboard_alert_mental_wellbeing_body">Deine Gefühlswelt nimmt Einfluss auf Deine Gedanken und Dein Verhalten. Finde Wege, dich öfter besser zu fühlen. </string>
+  <string name="recco_dashboard_alert_mental_wellbeing_title">Mentale Gesundheit</string>
+  <string name="recco_dashboard_alert_nutrition_body">Deine Gesundheit hängt auch davon ab, wie Du dich im Großen und Ganzen ernährst. So geht gesundes Essen ganz einfach: </string>
+  <string name="recco_dashboard_alert_nutrition_title">Essverhalten</string>
+  <string name="recco_dashboard_alert_physical_activity_body">Körperliche Aktivität wirkt sich positiv auf Körper und Geist aus - besonders, wenn die Bewegung Spaß macht! </string>
+  <string name="recco_dashboard_alert_physical_activity_title">Bewegung</string>
+  <string name="recco_dashboard_alert_sleep_body">Guter Schlaf ist das Fundament physischer und mentaler Gesundheit. Zum Glück kannst Du Deine Schlafqualität beeinflussen. </string>
+  <string name="recco_dashboard_alert_sleep_title">Schlaf</string>
+  <string name="recco_dashboard_explore_topic">Entdecken &amp; verstehen </string>
+  <string name="recco_dashboard_new_for_you">Neue Inhalte </string>
+  <string name="recco_dashboard_recommended_for_you_topic">Für dich empfohlen: <xliff:g id="section_name">%1$s</xliff:g>\n</string>
+  <string name="recco_dashboard_review_area">Bereich \nerkunden</string>
+  <string name="recco_dashboard_start_here">Schnell zum Erfolg </string>
+  <string name="recco_dashboard_trending">Beliebte Tipps</string>
+  <string name="recco_dashboard_unlock">Entsperren</string>
+  <string name="recco_dashboard_welcome_back_body">Mal sehen, was Deiner Gesundheit heute guttun könnte...</string>
+  <string name="recco_dashboard_welcome_back_title">Willkommen! </string>
+  <string name="recco_dashboard_you_may_also_like">Speziell für dich </string>
+  <string name="recco_error_connection_body">Atme tief durch und probiere es später nochmal. </string>
+  <string name="recco_error_connection_title">Keine Verbindung </string>
+  <string name="recco_error_generic_body">Das hätte nicht passieren dürfen. Lade die Seite neu. </string>
+  <string name="recco_error_generic_title">Entschuldigung! </string>
+  <string name="recco_error_reload">Neu laden</string>
+  <string name="recco_finish_button">Abschließen</string>
+  <string name="recco_next">Weiter</string>
+  <string name="recco_onboarding_go_to_dashboard">Zum Dashboard</string>
+  <string name="recco_onboarding_outro_body">Du hast Dein Profil erfolgreich eingerichtet. Schau dir jetzt an, was Recco dir zu bieten hat. </string>
```

Comment:
I noticed that some strings bring some extra trailing blank char in the German translation, that actually are not present in the English one. I dunno if this could be some sort of negative side-effect or not, maybe is something that needs to be inline in both files ?

Reply:
reported to translators, thanks @chemarubio, we wait to merge this PR for now 👍